lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -94,6 +95,10 @@ private function setFilters($parameters) {
 
     $filters['offset'] = $this->setOffset($filters['page'], $filters['count']);
 
+    if ($filters['exclude'] && !is_array($filters['exclude'])) {
```

Comment:
Not the cleanest, but the item here _needs_ to always be an array for Drupal to perform the `NOT IN` query condition.